pkg/receive/hashring.go

```diff
@@ -0,0 +1,68 @@
+package receive
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/improbable-eng/thanos/pkg/store/prompb"
+
+	"github.com/cespare/xxhash"
+)
+
+const sep = '\xff'
+
+// Hashring finds the correct host to handle a given time series
+// for a specified tenant.
+// It returns the hostname and any error encountered.
+type Hashring interface {
+	GetHost(string, *prompb.TimeSeries) (string, error)
```

Comment:
could we name these parameters here already, just so it's clear what the input and outputs are for this interface